qcut/docs/issues/caption-implementation/caption-system-architecture.md

```
@@ -0,0 +1,1429 @@
+exportAss(segments: TranscriptionSegment[]): string
+exportTtml(segments: TranscriptionSegment[]): string
+downloadCaptions(segments, format, filename): void
+```
+
+---
+
+## Data Flow
+
+### Transcription Workflow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)